src/coreclr/src/jit/codegencommon.cpp

```diff
@@ -6308,71 +6308,131 @@ void CodeGen::genZeroInitFrame(int untrLclHi, int untrLclLo, regNumber initReg,
         noway_assert(uCntBytes == 0);
 
 #elif defined(TARGET_XARCH)
-        /*
-            Generate the following code:
-
-                lea     edi, [ebp/esp-OFFS]
-                mov     ecx, <size>
-                xor     eax, eax
-                rep     stosd
-         */
+        noway_assert(compiler->getSIMDSupportLevel() >= SIMD_SSE2_Supported);
```

Comment:
This should be `assert`, not `noway_assert`.

Reply:
Changed